src/components/templates/Playa/Playa.tsx

```diff
@@ -429,7 +424,11 @@ const Playa = () => {
   const menuRef = useRef<HTMLDivElement | null>(null);
   const [, setRerender] = useState(0);
   const [shoutText, setShoutText] = useState("");
-  const [nowMs, setNowMs] = useState(new Date().getTime());
+  const [nowMs, setNowMs] = useState(Date.now());
```

Comment:
Policy note: this is good; just checked and these both use UTC. As long as we're avoiding any local time we are good.

Reply:
Agree, UTC AllTheThings!